scripts/test/ProcessManagement/TransitionAction/TransitionActionCheckCondition.t

```diff
@@ -0,0 +1,716 @@
+# --
```

Comment:
Please move this test to `scripts/test/ProcessManagement/TransitionAction/Base/TransitionActionCheckCondition.t`

Unfortunately, github doesn't let me edit so many lines at once and each change individually would be just work for both of us.
So I have uploaded the test for you here:

[TransitionActionCheckCondition.t.zip](https://github.com/znuny/Znuny/files/7285727/TransitionActionCheckCondition.t.zip)

Reply:
I committed your uploaded file, should be part of this PR now.
Thanks for your work & feedback!